src/components/metadata/Metadata.tsx

```diff
@@ -140,6 +141,7 @@ export const Metadata: React.FunctionComponent<Props> = (props) => {
   ] = useStore((store) => {
     return {
       manuscript: store.manuscript,
+      titles: store.titles,
```

Comment:
if you need the title in article-editor use it from prosemirror document like this: `store.doc.content.firstChild.textContent`